package.json

```diff
@@ -9,7 +9,8 @@
     "dev": "./bin/webpack-dev-server",
     "lint": "eslint 'app/javascript' --ext 'js,ts,jsx,tsx'",
     "typecheck": "tsc --noEmit --noErrorTruncation --pretty false --incremental false -p .",
-    "uml": "tplant -C --input app/javascript/types.ts --output types.svg"
+    "uml": "tplant -C --input app/javascript/types.ts --output types.svg",
```

Comment:
When I run this, I get an svg that tells me I need to accept a license somehow.  How?

Reply:
It said you accepted the license by opening the svg :)
And it was just blank because the file `app/javascript/types.ts` doesn't exist